_posts/2019-12-20-performance-now-conference-2019.md

```diff
@@ -0,0 +1,96 @@
+
+
+### Takeaway 2 - In Performance Engineering Data is Your Friend
+While progress in addressing the issues contributing to poor web performance has been slower than many might have envisaged, the same cannot be said for the tools available to help tackle the problem. The last 10 years have seen the proliferation and advancement in monitoring solutions and diagnostic tools that provide the insights needed to help improve web performance. Monitoring solutions like [NewRelic](https://newrelic.com/) and [AppDynamics](https://www.appdynamics.com/) can provide a 360 degree view of performance from infrastructure to application and [Real User Monitoring (RUM)](https://en.wikipedia.org/wiki/Real_user_monitoring). Tools like [Lighthouse](https://developers.google.com/web/tools/lighthouse) also provide valuable information aimed at improving site quality through analysis of pages against best practice criteria. All the talks at the conference drew on important insights obtained from these tools and it is true to say that web performance monitoring and diagnostics is no longer the dark art it once was. However, like anything - you need to ensure you choose the right tool for the right job.
+
+![Performance monitoring](/images/performance-now-conf-02.jpg)
+
+> “We need good top-level metrics” - Annie Sullivan
+
+Annie Sullivan, Software Engineer at Google, gave a talk entitled [“Lessons learned from performance monitoring in Chrome”](https://www.youtube.com/watch?v=ctavZT87syI). Annie talk covered performance metric and benchmark design, dealing with benchmark noise in the lab, and understanding the subtleties of RUM data. With all the data now available to us it’s sometimes difficult to see the wood for the trees. Annie told us about the properties that make up a good metrics and their associated use cases. While acknowledging that obtaining accurate insights from the metrics isn’t always easy, by focusing on the right metrics in the right setting we can gain a deeper understanding of performance problems from the lab through to real-user experience.
+
+Over the years I’ve made use of many tools to gain a better understanding of site performance - here are a few of my favourites that are free, easy to use, and provide a wealth of advice and insights :
+
+* [Sitespeed.io](https://www.sitespeed.io): Not mentioned at the conference but one of my favourites. Sitespeed.io is a set of Open Source tools that makes it easy to monitor and measure the performance of your web site. Sitespeed.io is the complete toolbox to test and  monitor your performance or checkout how your competition is doing.
+
+* [Webpagetest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```

Comment:
Consistent capitalisation - WebPageTest

Reply:
fixed

Reply:
As confusing as the naming is, the service name is capitalised WebPagetest - see [their about page](https://www.webpagetest.org/about).

```suggestion
* [WebPagetest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
```